packages/loaders/src/Progress.spec.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import Progress from './Progress';
+const { zdColorGreen600 } = require('@zendeskgarden/css-variables');
```

Comment:
Same as above. Could this use the standard `import from` syntax?

Reply:
Mhhh I probably copied this from somewhere, sorry about that.

Reply:
Ahh I copied it from an example, sorry.